src/application/ApplicationThunks.ts

```diff
@@ -338,12 +339,19 @@ export const loadApplicationConfigThunk = () => async (dispatch: any, getState:
     standaloneDashboardDatabase: 'dashboards',
     standaloneDashboardURL: '',
   };
-  try {
-    config = await (await fetch('config.json')).json();
-  } catch (e) {
-    // Config may not be found, for example when we are in Neo4j Desktop.
-    // eslint-disable-next-line no-console
-    console.log('No config file detected. Setting to safe defaults.');
+
+  // for now putting auth code here
```

Comment:
Lets use a global configuration here to call either the Hive logic or the default logic.

Perhaps something like:
```
if (HIVE_ENABLED) {
   // run the hive specific config retrieval
} else {
   // run the standard config retrieval
}
```